shared/chat/conversation/list-area/normal/index.native.tsx

```diff
@@ -188,7 +188,13 @@ class ConversationList extends React.PureComponent<Props> {
   render() {
     return (
       <ErrorBoundary>
-        <Box style={styles.container}>
+        <Box
+          style={Styles.collapseStyles([
+            styles.container,
+            this.props.hasStatus && {marginTop: 28},
+            this.props.hasPinnedMessage && {marginTop: 46},
```

Comment:
these are kind of magic numbery, but they're based on the rendered height of pinned messages and banners on mobile (which don't have nice constants to use, afaict)